src/components/atoms/HomeArrow.vue

```diff
@@ -1,13 +1,50 @@
 <template>
-	<div class="home__arrow-outline">
+	<div v-on:click="traverse(arrow, containers)" class="home__arrow-outline">
```

Comment:
I suggest you to have a carousel functionality in organisms since carousel its a big component that have a lot of things inside and should have carousel logic there as well
but arrow is an atom, atoms should be just a piece of your app that can be placed anywhere (example, this arrow in box can be used in navigation and carousel, you only need to specify variants dark/light)

Reply:
all big functionality should go to organisms, small functionality in functionality (dropdown or smth like that) and no functionality (except maybe hover effects) in atoms